logfire/_internal/main.py

```diff
@@ -154,6 +154,7 @@ def _span(
         _tags: Sequence[str] | None = None,
         _span_name: str | None = None,
         _level: LevelName | int | None = None,
+        _links: Sequence[trace_api.Link] = (),
```

Comment:
```suggestion
        _links: Sequence[trace_api.Link | trace_api.SpanContext | trace_api.Span | ReadableSpan | LogfireSpan] = (),
```

Making users import and construct link objects feels clunky. They should easily be able to pass any of these things, from which we can get a span context and construct a link, assuming they don't need attributes. We could maybe even allow a typed dict or a 2-tuple. None of this is important now, and it might clutter things up, just a thought.

Reply:
I like an analogous API to the `add_link`: `_links: Sequence[tuple[trace_api.SpanContext, otel_types.Attributes]]`